sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -150,7 +165,20 @@ public HttpPipeline getHttpPipeline() {
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PagedFlux<String> listRelationships(String digitalTwinId, String relationshipName) {
```

Comment:
The list API still needs work, I'll fix this in the relationships impl PR #14527